lib/utils/utils.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = {
+  isAngularModuleCall: function (node) {
+    return (node.object.name === 'angular' && node.property.name === 'module')
+  },
+
+  isClassEmpty: function (classNode) {
+    const methods = classNode.body.body;
+    return !methods || methods.length === 0;
+  },
+
+  getConstructor: function (classNode) {
+    return _.find(classNode.body.body, (method) => method.type === 'MethodDefinition' && method.kind === 'constructor');
+  },
+
```

Comment:
while both of these two utilities is more general because you pass in the specific directivePrologue you are looking for (i.e., ngInject) in reality how many directivePrologue's are we going to be testing for. By hardcoding in 'ngInject' and changing the functon name to be more explicit I think the code becomes marginally more readable, which IMO is worth it. Example function name : `constructorContainsNginject` , `inlineDirectiveFuncContainsNginject` ?